### PR TITLE
Remove manual memory management from app_ofline

### PR DESCRIPTION
**Pull request by pakrym**
And add tracing

### CODE REVIEW

File: src/AspNetCoreModuleV2/AspNetCore/Inc/appoffline.h [hunk 1]
```diff
@@ -14,92 +16,64 @@ class APP_OFFLINE_HTM
...
+            if (GetLastError() == ERROR_FILE_NOT_FOUND)
             {
-                fResult = FALSE;
+                return false;
             }
 
             // This Load() member function is supposed be called only when the change notification event of file creation or file modification happens.
             // If file is currenlty locked exclusively by other processes, we might get INVALID_HANDLE_VALUE even though the file exists. In that case, we should return TRUE here.
-            goto Finished;
+            LOG_LAST_ERROR_IF(true);
```
Comment (jkotalik) [inline src/AspNetCoreModuleV2/AspNetCore/Inc/appoffline.h]: This is a bit weird. Add a comment saying why you are doing this.

File: src/AspNetCoreModuleV2/AspNetCore/Inc/appoffline.h [hunk 2]
```diff
@@ -14,92 +16,64 @@ class APP_OFFLINE_HTM
...
         }
 
-        if (pszBuff != NULL)
-        {
-            delete[] pszBuff;
-            pszBuff = NULL;
-        }
-
-        return fResult;
+        return true;
```
Comment (jkotalik) [inline src/AspNetCoreModuleV2/AspNetCore/Inc/appoffline.h]: Lot of return true here. Didn't see an easy way to reduce it besides nesting ifs.

**Comment by jkotalik** [comment]
Build is failing; missing include?

**Comment by pakrym** [comment]
I'll close this until @pan-wang  work is in.

**Comment by pan-wang** [comment]
I removed app-offline class. You can close this one

**Pull request closed without merging**